build.sbt

```diff
@@ -238,6 +238,31 @@ lazy val semconv = crossProject(JVMPlatform, JSPlatform, NativePlatform)
     name := "otel4s-semconv"
   )
 
+lazy val `sdk-common` = crossProject(JVMPlatform, JSPlatform, NativePlatform)
+  .crossType(CrossType.Pure)
+  .in(file("sdk/common"))
+  .dependsOn(`core-common`)
+  .settings(
+    name := "otel4s-sdk-common",
+    libraryDependencies ++= Seq(
+      "org.typelevel" %%% "cats-effect-kernel" % CatsEffectVersion,
+      "org.typelevel" %%% "cats-mtl" % CatsMtlVersion,
+    ),
+  )
+  .settings(munitDependencies)
+
+lazy val `sdk-trace` = crossProject(JVMPlatform, JSPlatform, NativePlatform)
```

Comment:
Oh I forgot, you also need to add these to the "root aggregate" somewhere higher up in the build. I think that's why CI appears to be passing, because it's not actually compiling these sources yet 😅